.github/workflows/main.yml

```diff
@@ -104,14 +104,23 @@ jobs:
             //software/ai/hl/stp/play/...    \
             //software/ai/hl/stp/tactic/... 
 
-      
       - name: Upload simulated test artifacts
         uses: actions/upload-artifact@v2
         with:
           name: simulated-test-outputs
           # we only care about the replay logs from the STP tests
           path: |
             src/bazel-out/k8-fastbuild/testlogs/software/ai/hl/stp
+
+      - name: Simulated Test (Check for errors)
+        # Run Simulated Tests a second time to check for failures
+        run: |
+          cd src
+          bazel test --show_timestamps   \
+            //software/simulated_tests/...   \
```

Comment:
LGTM! Question: does the \ make the tests run consecutively?

Reply:
the `\` splits a single command onto multiple lines and it helps with readability